### PR TITLE
chore: fix memcached pipeline test

### PR DESCRIPTION
**Pull request by dranikpg**
Fixed failing test

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -172,7 +172,7 @@ jobs:
           dfly-executable: dragonfly
           run-only-on-ubuntu-latest: true
           build-folder-name: build
-          filter: ${{ matrix.build-type == 'Release' && 'not slow' || '(not slow) and (not opt_only)' }}
```
Comment (romange) [inline .github/workflows/ci.yml:175]: is it part of the fixes?

File: .github/workflows/regression-tests.yml [hunk 2]
```diff
@@ -42,11 +42,7 @@ jobs:
           dfly-executable: dragonfly
           gspace-secret: ${{ secrets.GSPACES_BOT_DF_BUILD }}
           build-folder-name: build
-          # This expression serves as a ternary operator, i.e. if the condition holds it returns
```
Comment (romange) [inline .github/workflows/regression-tests.yml:45]: what have been fixed here?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dranikpg) [reply .github/workflows/ci.yml:175]: Yes

File: .github/workflows/regression-tests.yml (hunk 2 above)
Comment (dranikpg) [reply .github/workflows/regression-tests.yml:45]: I added dbg_only instead of NOT_EXISTING_MARKER

Comment (dranikpg) [reply .github/workflows/regression-tests.yml:45]: Dbg_only because the release build is too fast

**Review by romange — APPROVED** [review]

**Pull request merged**